### ISSUE TITLE
Verify output with the parity bits

### ISSUE THREAD

**Issue by surajkadapa** [surajkadapa/hamming_code_1#2]
Make a circuit that takes the generated parity bits and the input data and the output (data) is not changed even if ONLY ONE of the bits is changed (parity or data).

**Comment by RishavSinha20** [comment]
can i work on this?

**Comment by surajkadapa** [comment]
@RishavSinha20 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by surajkadapa** [comment]
@RishavSinha20 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by surajkadapa** [comment]
@RishavSinha20 time limit is increased by 35 mins

**Comment by umarahmed10** [comment]
can i get assigned

**Comment by surajkadapa** [comment]
@RishavSinha20 reassigning because time is out.

**Comment by surajkadapa** [comment]
@umarahmed10 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by 411sst** [comment]
can i get assigned if this is still open?

**Comment by surajkadapa** [comment]
@ssether04 Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by Shreevardhancoder** [comment]
can i be assigned ?

**Comment by surajkadapa** [comment]
@Shreevardhancoder Yes. The time limit is 1 hour. If not, others can ask to be assigned.

**Comment by surajkadapa** [comment]
@Shreevardhancoder any update?
